src/pf_driver/src/pf/pf_interface.cpp

```diff
@@ -22,6 +22,11 @@ bool PFInterface::init(std::shared_ptr<HandleInfo> info, std::shared_ptr<ScanCon
   info_ = info;
   params_ = params;
 
+  params_->active_timesync.init(config_->timesync_period, config_->timesync_offset_usec,
+                                (config_->timesync_averaging == TIMESYNC_AVERAGING_REGRESSION));
```

Comment:
timesync_averaging is initialized with 0 (TIMESYNC_AVERAGING_MEAN) so the following suggestion should also work and is more clear. Because in the current state the result of a comparison (bool) is the 3rd parameter of the init function which is really an int. See also the next init() call for passive_timesync
```suggestion
                                config_->timesync_averaging);
```

Reply:
You're right, the bool was created to satisfy the previous init() that was expecting a bool, but now an int should be passed. Corrected in 3f040f8 (and reformatted with clang-format)